src/components/calculator.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Keys from './operator';
+import calculate from '../logic/Calculate';
+
+class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { total: 0, next: null, operation: null };
+  }
+
+componentDidMount = () => {
+  this.setState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+}
+
+render() {
+  const handleEvent = (e) => {
+    const oprObject = calculate(this.state, e.target.textContent);
+    this.setState(oprObject);
+  };
+
+  const { total, operation, next } = this.state;
+  const oprnd = operation === '%' ? 'mod' : operation;
+  let result = '';
+  if (total) {
+    result = `${total} ${oprnd || ''} ${next || ''}`;
+  } else if (next) {
+    result = `${next} ${oprnd || ''}`;
+  }
+  return (
+    <div className="wrapper">
+      <div className="screen">{result || 0}</div>
+      <Keys styles="btn light-gray" handleEvent={(e) => handleEvent(e)} val="AC" />
+      <Keys styles="btn light-gray" handleEvent={(e) => handleEvent(e)} val="%" />
+      <Keys styles="btn light-gray" handleEvent={(e) => handleEvent(e)} val="+/-" />
+      <Keys styles="btn orange" handleEvent={(e) => handleEvent(e)} val="/" />
+      <Keys styles="btn" handleEvent={(e) => handleEvent(e)} val="7" />
```

Comment:
- [ ] Good job by implementing the all basics operations for your Calculator app. But, for now, the division operation doesn't work as expected. Once we try to proceed with the division operation this error below appears:
![image](https://user-images.githubusercontent.com/71874425/171696980-03616863-f940-45df-ab1c-cc11910d7ffd.png)
To solve this, Instead of using slash `"/"` for the division symbol I'll suggest you use this symbol `"÷"`  As assigned to the operation in the `Operate.js` component.